templates/index.php

```diff
@@ -9,26 +9,18 @@
 ?>
 
 <div id="app">
-	<div id="app-navigation" class="icon-loading">
-		<ul>
-			<li id="mail-new-message-fixed">
-			<input type="button" id="mail_new_message" class="icon-add"
-				value="<?php p($l->t('New message')); ?>" style="display: none">
-			</li>
-			<li id="mail-new-message-list">
-				<div id="folders"></div>
-			</li>
-		</ul>
+	<div id="app-navigation">
+		<ul id="app-navigation-entries"></ul>
 		<div id="app-settings">
 			<div id="app-settings-header">
 				<button class="settings-button"
 					data-apps-slide-toggle="#app-settings-content"></button>
 			</div>
-			<div id="app-settings-content"> </div>
+			<div id="app-settings-content"></div>
 		</div>
 	</div>
 	<div id="app-content">
-		<div id="mail_messages" class="icon-loading">
+		<div id="mail_messages">
```

Comment:
> Is there a reason why some id's are dash separated and others are with underscore? It's would suggest to stick with the one or the other version ...
> 
> Regards
> 
> Ingo

I don't think so. We more or less stick to dashes AFAIK, but some 'old' id/classes using underscore have not been changed yet because changing could potentially breaking something (they are used both in JS and CSS). Therefore, if you add something new, use dashes ;-)